docs/src/samples/cluster-example-logical-source.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: cluster-example
+spec:
+  instances: 1
+
+  imageName: ghcr.io/cloudnative-pg/postgresql:13
+
+  storage:
+    size: 1Gi
+
+  bootstrap:
+    initdb:
+      postInitApplicationSQL:
+        - CREATE TABLE numbers (i SERIAL PRIMARY KEY, m INTEGER)
+        - INSERT INTO numbers (m) (SELECT generate_series(1,10000))
+        - ALTER TABLE numbers OWNER TO app;
+        - CREATE TABLE numbers_two (i SERIAL PRIMARY KEY, m INTEGER)
+        - INSERT INTO numbers_two (m) (SELECT generate_series(1,10000))
+        - ALTER TABLE numbers_two OWNER TO app;
+        - CREATE TABLE numbers_three (i SERIAL PRIMARY KEY, m INTEGER)
+        - INSERT INTO numbers_three (m) (SELECT generate_series(1,10000))
+        - ALTER TABLE numbers_three OWNER TO app;
+
+  # This is needed to enable users to create the publication while
+  # they are connected from the destination. Like in:
```

Comment:
again, "while they are connected".
The commands below can be executed from the plain shell, you don't need to be connected before.
I think what is meant is something like
"to create the publication by issuing commands to be executed on the destination cluster"